MaterialDesignExtensions/MaterialDesignExtensions.csproj

```diff
@@ -1,320 +1,79 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{809632DA-5EB8-4EE8-AD71-57239701550C}</ProjectGuid>
-    <OutputType>library</OutputType>
-    <RootNamespace>MaterialDesignExtensions</RootNamespace>
-    <AssemblyName>MaterialDesignExtensions</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <WarningLevel>4</WarningLevel>
-    <TargetFrameworkProfile />
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
+    <!--<TargetFrameworks>netcoreapp3.1;net48</TargetFrameworks>-->
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Not using multiple targets breaks many applications that cannot be updated because their third party dependencies do not support .NET Core. That's why I keep on building DLLs for .NET Framework 4.5.